### DISCUSSION TITLE
keyball44LEDランド剥がれ(左手noBallSide-no1, 右手ballSide-no18)

### DISCUSSION THREAD

**Discussion by hm-nnRbioso** [Yowkees/keyball#268] [Q&A]
LEDのランドが剥がれてしまいまして、対処方法等を質問させてください。初歩的な質問もあり、恐縮です。

 - noBallSide-no1についてはDIN, GNDと思われる2つのランドが剥がれました。（写真中、赤・青星の印をつけた箇所です）それぞれどちらに繋げばよろしいでしょうか？
![左手noBallSide-no1](https://github.com/Yowkees/keyball/assets/89882827/9201d91f-71c5-4dc1-b23c-d2b6a8bef731)
 - ballSide-no18についてはDIN, DOUTと思われる2つのランドが剥がれました。（写真中、赤・青星の印をつけた箇所です）18DIN->17DOUT, 18DOUT->19DINへ繋ぐので合っていますでしょうか？
![右手ballSide-no18](https://github.com/Yowkees/keyball/assets/89882827/fadb1165-f2f0-4a6f-bad6-48744bcebf57)
 - GND, DIN, DOUTの見分け方ですが、黒丸がGND,その上がDIN, 黒丸の左右水平に反対側がDOUT、という見分け方で合っていますでしょうか？
 - 今回エナメル線を使ってランド剥がれを修復しようとしていますが、エナメル線をどのように這わせたら良いかわからず、もし何かアドバイス頂けるようでしたらお願いできませんでしょうか？（他の部品パーツに触れてしまっても直線で繋いで良いのか、部品パーツを避けて基盤状を這わせるのが良いのか、というあたりが気になっています）
 - LEDパーツが加加熱で破損しLEDパーツを交換する際にランドが剥がれないよううまく除去する方法があれば、教えてくださいますでしょうか？

いくつも申し訳ありませんが、何卒よろしくお願いいたします！

**Comment by Yowkees** [comment]
ご連絡ありがとうございます。
まず１番LEDの修復ですが、下記画像の赤星は赤丸か赤四角のどちらか半田付けしやすいほうに接続してください。同様に青星は青丸か青四角のどちらか接続しやすいほうに接続してください。
![KB44L_LEDfix](https://github.com/Yowkees/keyball/assets/84209772/7bff1143-45b3-44bf-b03f-389a1df294ad)

**Comment by Yowkees** [comment]
ballSide-no18についてはDIN, DOUTと思われる2つのランドが剥がれました。（写真中、赤・青星の印をつけた箇所です）18DIN->17DOUT, 18DOUT->19DINへ繋ぐので合っていますでしょうか？
■はい合っています。

GND, DIN, DOUTの見分け方ですが、黒丸がGND,その上がDIN, 黒丸の左右水平に反対側がDOUT、という見分け方で合っていますでしょうか？
■はい。こちらもその通りです。

今回エナメル線を使ってランド剥がれを修復しようとしていますが、エナメル線をどのように這わせたら良いかわからず、もし何かアドバイス頂けるようでしたらお願いできませんでしょうか？（他の部品パーツに触れてしまっても直線で繋いで良いのか、部品パーツを避けて基盤状を這わせるのが良いのか、というあたりが気になっています）
■エナメルは絶縁体でして、エナメル線はエナメルに覆われているのでゴム被覆同様に電気を流しません。ですので、線が途中で他の部品に触れても大丈夫です。エナメル線は端を、半田をつけた小手先で熱するとエナメルが解けて銅線が出てくるので、ニッパで被覆をはがす作業が不要で扱いやすいです。
できるだけ基板に沿うようにクセをつけて目的の個所までくねくね配線してください。

LEDパーツが加加熱で破損しLEDパーツを交換する際にランドが剥がれないよううまく除去する方法があれば、教えてくださいますでしょうか？
■半田ごて２本で両端から熱しながら、２本の小手で挟んで持ち上げる方法が確実です。私はいつもこの方法です。
はんだごてが２本必要な場面は結構あるので、安価なものでも良いので１本余分に持っておくと便利です。

以上、お手数ですがご対応お願いいたします。

**Reply by hm-nnRbioso** [reply]
たくさん教えていただき、本当にありがとうございました！無事LED全て点灯しました！

最後に一点教えていただけますでしょうか。
ランドが剥がれているせいか、エナメル線にもLEDの足にも綺麗にはんだが定着せず、写真のようにとても汚い感じになってしまいました。綺麗に修正するコツがあれば教えていただけますでしょうか？
![IMG_4175](https://github.com/Yowkees/keyball/assets/89882827/957d2acb-719e-402a-a9a9-1c246eb39344)

P.S.質問とは関係ありませんが、とても楽しい体験と製品をありがとうございました！！！このキーボードを使って楽しく仕事できそうです！

**Reply by Yowkees** [reply]
お褒めいただき、大変光栄です。ありがとうございます。

エナメル線ですが、表面がエナメル塗料で覆われているので、そのままでは半田付けできません。
お写真の状態はかろうじてくっついているだけかと。

まずエナメル線の端をはんだをたっぷりつけた小手先に突っ込んで、端っこだけエナメルを溶かします。中の導線だけが残りまして、導線には半田が付きます。ご対応お願いいたします。
